api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/processor/DetermineEntitlementMessageProcessor.java

```diff
@@ -59,26 +59,26 @@ public MessageStatus processMessage(Message message) {
         PaymentCycle currentPaymentCycle = messageContext.getCurrentPaymentCycle();
         PaymentCycle previousPaymentCycle = messageContext.getPreviousPaymentCycle();
 
-        EligibilityAndEntitlement eligibility = eligibilityService.determineEligibilityForExistingClaimant(
+        EligibilityAndEntitlementDecision decision = eligibilityService.determineEligibilityAndEntitlementForExistingClaimant(
                 claimant,
                 currentPaymentCycle.getCycleStartDate(),
                 previousPaymentCycle);
 
         //TODO HTBHF-1296 - update ClaimStatus from ACTIVE to PENDING_EXPIRY if Claimant is no longer eligible.
-        updateAndSaveCurrentPaymentCycle(currentPaymentCycle, eligibility);
+        updateAndSaveCurrentPaymentCycle(currentPaymentCycle, decision);
 
-        if (eligibility.getEligibilityStatus() == ELIGIBLE) {
+        if (decision.getEligibilityStatus() == ELIGIBLE) {
```

Comment:
Minor - is it worthwhile adding an `isEligibile()` method to `EligibilityAndEntitlementDecision`?

Reply:
This is the only place it would be used, so I might leave it for now.